### ISSUE TITLE
Integrate OMADA API as well

### ISSUE THREAD

**Issue by awlx** [freifunkMUC/unifi_respondd#27]
There is a simple API example here:

https://github.com/ghaberek/omada-api

**Comment by awlx** [comment]
Complete API reference for controller v5
[Omada_SDN_Controller_V5.0.15 API Document.html.zip](https://github.com/freifunkMUC/unifi_respondd/files/8416411/Omada_SDN_Controller_V5.0.15.API.Document.html.zip)

**Comment by T0biii** [comment]
first fork to get this running:
https://github.com/T0biii/unifi_respondd/commits/omada-fork
hope to find some time soon ;)
help always appreciated

**Comment by Inrin** [comment]
Software Controller here: https://www.tp-link.com/de/support/download/omada-software-controller/#Controller_Software

**Comment by Inrin** [comment]
@awlx do you target any specific minimum version?
Else I guess v5.0.30 as minimum?

**Comment by awlx** [comment]
Yes, 5.0.x as minimum works most of us are on latest and greatest anyway :).

**Comment by T0biii** [comment]
see https://github.com/freifunkMUC/omada_respondd

**Comment by T0biii** [comment]
https://github.com/freifunkMUC/omada_respondd
